ruby/vampires.rb

```diff
@@ -0,0 +1,68 @@
+def foodMethod (food)

```

Comment:
no space between method name and param.

also ruby is snake case `food_method` not camel case `foodMethod`